src/main/java/org/spongepowered/api/data/type/BodyPart.java

```diff
@@ -25,9 +25,10 @@
 package org.spongepowered.api.data.type;
 
 import org.spongepowered.api.CatalogType;
+import org.spongepowered.api.text.translation.Translatable;
 import org.spongepowered.api.util.annotation.CatalogedBy;
 
 @CatalogedBy(BodyParts.class)
-public interface BodyPart extends CatalogType {
+public interface BodyPart extends CatalogType, Translatable {
```

Comment:
I will change that. IMO the all concerned the same API so I did not separate them.

* [ ] Split PR (Translatable)